core/server/data/schema/commands.js

```diff
@@ -49,6 +49,9 @@ function addTableColumn(tableName, table, columnName, columnSpec = schema[tableN
     if (Object.prototype.hasOwnProperty.call(columnSpec, 'index') && columnSpec.index === true) {
         column.index();
     }
+    if (Object.prototype.hasOwnProperty.call(columnSpec, 'index') && _.isArray(columnSpec.index)) {
+        table.index(columnSpec.index);
+    }
```

Comment:
This feels pretty hacky but I wanted the quickest-workable-thing to support composite indexes during db init. Any thoughts on a quick but less hacky option?